### PR TITLE
Updated docker container and added test for lung cancer scenario

### PR DESCRIPTION
**Pull request by jaigoradia**

### CODE REVIEW

**Review by csandersdev — APPROVED** [review]
LGTM

**Review by ToddHicks — APPROVED** [review]
Looks good, we still have a few more changes to go, but we are well on our way!

**Review by msargentibm — APPROVED** [review]

File: tests/docker/cloudInit.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#
+#
+# note ...
+# must run as " . ./script.sh"
+# that is, if you don't start with a dot then the
+# environment export does not work.
```
Comment (dkwasny-ibm) [inline tests/docker/cloudInit.sh:6]: Is this comment intended for users running this script directly?
It looks like the only place this script is uses here doesn't use the source (dot) notation.

File: tests/docker/taurus-bridge.dockerfile [hunk 2]
```diff
@@ -2,4 +2,19 @@ FROM blazemeter/taurus
...
+
+RUN  curl -fsSL https://clis.cloud.ibm.com/install/linux | bash && \
+    ibmcloud plugin install container-service -f && \
+    curl -sL "https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl" > /usr/local/bin/kubectl && \
+    chmod 755 /usr/local/bin/kubectl && \
+    kubectl version --client
+
+COPY cloudInit.sh /
+
+#RUN /cloudInit.sh pziZ4lJxZY_pQCgByMPWISjxy5AN_9_L875S-mtGjqX6 CDT_CommOps_Quick_Cluster_01 CDT_CommOps_Quick_Cluster_01_RG cdt-commops-quickteam01-ns-01
```
Comment (dkwasny-ibm) [inline tests/docker/taurus-bridge.dockerfile:18]: Does this comment contain a legit API key?

**Review by dkwasny-ibm — CHANGES_REQUESTED** [review]
Minor questions.

File: tests/docker/cloudInit.sh (hunk 1 above)
Comment (jaigoradia) [reply tests/docker/cloudInit.sh:6]: Removed comment from file.

File: tests/docker/taurus-bridge.dockerfile [hunk 3]
```diff
@@ -2,4 +2,19 @@ FROM blazemeter/taurus
 
 RUN pip3 install numpy
 
-RUN pip3 install python-javabridge
\ No newline at end of file
+RUN pip3 install python-javabridge
+
+ARG KUBE_VERSION=1.16.1
+ENV API_KEY=“set-me”
```
[the remark this answers is not in the document]
Comment (jaigoradia) [reply tests/docker/taurus-bridge.dockerfile:8]: Changed quotes to be consistent.

File: tests/docker/taurus-bridge.dockerfile (hunk 2 above)
Comment (jaigoradia) [reply tests/docker/taurus-bridge.dockerfile:18]: Removed comment from file.

**Review by rickjstevens — APPROVED** [review]

File: tests/docker/taurus-bridge.dockerfile [hunk 4]
```diff
@@ -2,4 +2,19 @@ FROM blazemeter/taurus
...
+    ibmcloud plugin install container-service -f && \
+    curl -sL "https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl" > /usr/local/bin/kubectl && \
+    chmod 755 /usr/local/bin/kubectl && \
+    kubectl version --client
+
+COPY cloudInit.sh /
+
+#RUN /cloudInit.sh pziZ4lJxZY_pQCgByMPWISjxy5AN_9_L875S-mtGjqX6 CDT_CommOps_Quick_Cluster_01 CDT_CommOps_Quick_Cluster_01_RG cdt-commops-quickteam01-ns-01
+
+ENTRYPOINT ["sh", "-c", "/cloudInit.sh ${API_KEY} CDT_CommOps_Quick_Cluster_01 CDT_CommOps_Quick_Cluster_01_RG cdt-commops-quickteam01-ns-01; bzt -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
```
Comment (jillrdoty) [inline tests/docker/taurus-bridge.dockerfile:18]: I have some mixed feelings about putting CDT specific things into our open project.  Will add as a topic to discuss with the team.

**Review by dkwasny-ibm — APPROVED** [review]

File: tests/src/main/python/TestDriver_Json.py [hunk 5]
```diff
@@ -63,9 +63,14 @@ def setup_class(self):
         global engineWrapper
         testWrapper = baseDir+os.environ['TEST_WRAPPER']
         cohortEngine = baseDir+os.environ['COHORT_ENGINE']
-        javabridge.start_vm(run_headless=True, class_path=javabridge.JARS + [cohortEngine, testWrapper]) # Start the JVM with modified classpath.
+        #  -Xms512M -Xmx1G -Djavax.net.ssl.trustStore=config/trustStore.pkcs12 -Djavax.net.ssl.trustStorePassword=change-password -Djavax.net.ssl.trustStoreType=pkcs12 
+        print("testwrapper: "+ testWrapper)
+        print("cohortEngine: "+ cohortEngine)
+        print("fhirServer: "+ os.environ['DATA_FHIR_SERVER_DETAILS'])
```
Comment (ToddHicks) [inline tests/src/main/python/TestDriver_Json.py:69]: The comment and these print statements should be removed.

File: tests/src/main/python/TestDriver_Json.py [hunk 6]
```diff
@@ -63,9 +63,14 @@ def setup_class(self):
...
+        javabridge.start_vm(args=["-Xms512M", "-Xmx1G", "-Djavax.net.ssl.trustStore=cohort-cli/config/trustStore.pkcs12", "-Djavax.net.ssl.trustStorePassword=change-password", "-Djavax.net.ssl.trustStoreType=pkcs12"], run_headless=True, class_path=javabridge.JARS + [cohortEngine, testWrapper]) # Start the JVM with modified classpath.
+        #javabridge.start_vm(run_headless=True, class_path=javabridge.JARS + [cohortEngine, testWrapper]) # Start the JVM with modified classpath.
```
Comment (ToddHicks) [inline tests/src/main/python/TestDriver_Json.py:71]: Comment should be removed.

File: tests/src/main/resources/config/local-ibm-fhir.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+    "@class": "com.ibm.cohort.engine.IBMFhirServerConfig",
+    "endpoint": "https://localhost:9443/fhir-server/api/v4",
+    "password": "change-password",
+    "tenantId": "test-fvt",
```
Comment (ToddHicks) [inline tests/src/main/resources/config/local-ibm-fhir.json:5]: Do we have any reason to be concerned about this being published?

**Review by ToddHicks — APPROVED** [review]
If the lines that I commented on needing deletion are removed this looks fine to me.

File: tests/src/main/python/TestDriver_Json.py (hunk 5 above)
Comment (jaigoradia) [reply tests/src/main/python/TestDriver_Json.py:69]: Removed Comment and Print statements.

File: tests/src/main/python/TestDriver_Json.py (hunk 6 above)
Comment (jaigoradia) [reply tests/src/main/python/TestDriver_Json.py:71]: Done.

File: tests/src/main/resources/config/local-ibm-fhir.json (hunk 7 above)
Comment (jaigoradia) [reply tests/src/main/resources/config/local-ibm-fhir.json:5]: Corey says those are well-known values published in the IBM FHIR user guide.

Comment (jaigoradia) [reply tests/src/main/resources/config/local-ibm-fhir.json:5]: Here are the docs: https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide

**Pull request merged**